ansible/roles/ocp4-workload-infra-nodes/defaults/main.yml

```diff
@@ -3,5 +3,11 @@ become_override: False
 ocp_username: opentlc-mgr
 silent: False
 
-_infra_node_replicas: 3
-_infra_node_instance_type: m4.4xlarge
\ No newline at end of file
+_infra_node_replicas: 1
```

Comment:
Is this going to cause a problem with the number of replicas needed for some of the infra components?

Reply:
You can always make it bigger... Also we can move logging to dedicated nodes now.

Reply:
You can also make the instance types bigger. These are just defaults.